support-frontend/assets/pages/[countryGroupId]/events/router.tsx

```diff
@@ -0,0 +1,21 @@
+import ophan from 'ophan';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { renderPage } from 'helpers/rendering/render';
+import { geoIds } from 'pages/geoIdConfig';
+import { Events } from './events';
+
+ophan.init();
```

Comment:
@paul-daniel-dempsey I've moved this to the router, which felt more appropriate given this is where the entry point of the page actually is.

Reply:
Can see this initialisation is part of the call `setUpTrackingAndConsent()` which is placed on most pages including the `supporterPlusRouter.tsx`